templates/500.html

```diff
@@ -1,25 +1,11 @@
-{% extends 'base_layout.html' %}
+{% extends '/_error.tmpl' %}
```

Comment:
```suggestion
{% extends '/partial/_error.tmpl' %}
```
Shouldn't match the PATH for the `404` page?

Reply:
It absolutely should, thanks for catching!